eightpoints/guzzle-bundle/7.0/config/packages/eight_points_guzzle.yaml

```diff
@@ -0,0 +1,14 @@
+eight_points_guzzle:
+    clients:
+        api_payment:
+            base_url: "http://api.domain.tld"
```

Comment:
Remove

Reply:
Better to keep here some URL. Probably one 100% not active or to write `http://target.url`. It's better to enforce users to write host here, not in controllers and it's better to find proper way to do that